backend/app/api/v1/endpoints/cpt/cptJobs.py

```diff
@@ -0,0 +1,134 @@
+import json
+from fastapi import Response
+import pandas as pd
+from datetime import datetime, timedelta, date
+from fastapi import APIRouter
+from .maps.ocp import ocpMapper
+from fastapi.param_functions import Query
+
+router = APIRouter()
+
+products = {
+            "ocp": ocpMapper,
+           }
+
+
+response_example ={
```

Comment:
Duplicate snippet found in ocpJobs, can we define it in one place and import it here?

Reply:
Thanks, actually I need to update this to the new format good catch